store/postgres/src/store.rs

```diff
@@ -294,15 +294,27 @@ impl Store {
         }
     }
 
-    /// Applies a set operation in Postgres.
-    fn apply_set_operation(
+    /// Applies a set operation with the given block as the event source.
+    fn apply_set_operation_with_block(
         &self,
         conn: &PgConnection,
         operation: EntityOperation,
         block_ptr_to: EthereumBlockPointer,
-    ) -> Result<usize, Error> {
+    ) -> Result<(), Error> {
+        self.apply_set_operation_with_conn(conn, operation, &block_ptr_to.hash_hex())
+    }
+
+    /// Applies a set operation in Postgres.
+    fn apply_set_operation_with_conn(
+        &self,
+        conn: &PgConnection,
+        operation: EntityOperation,
+        op_event_source: &str,
+    ) -> Result<(), Error> {
```

Comment:
Any particular reason (other than that we're not currently using this) not to return the number of rows here?

Reply:
We don't use it anywhere, and it's not something we want to return in the public interface since we try to keep the `Store` trait backend-agnostic while the number of affected rows is specific to relational dbs.

Reply:
Yep, ok, let's do it this way then. 😄